modern/src/reports/ReplayPage.js

```diff
@@ -41,58 +62,138 @@ const TimeLabel = ({ children, open, value }) => (
+                  <Grid item>
+                    <IconButton onClick={() => history.push('/')}>
+                      <ArrowBackIcon />
+                    </IconButton>
+                  </Grid>
+                  <Grid item xs>
+                    <Typography className={classes.replay} color="primary">
+                      {t('reportReplay')}
+                    </Typography>
+                  </Grid>
+                  {isPlaying && (
+                    <Grid item>
+                      <IconButton onClick={() => setIsPlaying(false)}>
+                        <SettingsIcon />
+                      </IconButton>
+                    </Grid>
+                  )}
+                </Grid>
+              </Toolbar>
+            </Paper>
+          </Grid>
+          <Grid item>
+            {isPlaying ? (
+              <Paper className={classes.sliderContainer}>
+                <Grid container direction="column" alignItems="center">
+                  <Grid item>
+                    <span className={classes.title}>{deviceName}</span>
+                  </Grid>
+                  <Grid item style={{ width: '100%' }}>
+                    <Slider
+                      max={positions.length - 1}
+                      step={null}
+                      marks={positions.map((_, index) => ({ value: index }))}
+                      value={index}
+                      onChange={(_, index) => setIndex(index)}
+                      valueLabelDisplay="auto"
+                      valueLabelFormat={(i) => (i < positions.length ? formatPosition(positions[i], 'fixTime') : '')}
+                      ValueLabelComponent={TimeLabel}
+                    />
+                  </Grid>
+                  <Grid item container justifyContent="space-between" alignItems="center">
+                    <Grid item xs={2}>{`${index}/${positions.length}`}</Grid>
+                    <Grid item xs={2}>
+                      <IconButton>
+                        <FastRewindIcon />
+                      </IconButton>
+                    </Grid>
+                    <Grid item xs={2}>
+                      <IconButton>
+                        <PlayArrowIcon />
+                      </IconButton>
+                    </Grid>
+                    <Grid item xs={2}>
+                      <IconButton>
+                        <FastForwardIcon />
+                      </IconButton>
+                    </Grid>
+                    <Grid item xs>{formatPosition(positions[index], 'fixTime')}</Grid>
+                  </Grid>
+                </Grid>
+              </Paper>
+            ) : (
+              <Paper elevation={3} className={classes.reportFilterContainer} square>
+                <ReportFilter handleSubmit={handleSubmit} fullScreen showOnly>
+                  <Grid item xs={6}>
+                    <TextField
+                      fullWidth
+                      label={t('reportPlaybackPerMinute')}
```

Comment:
Is this available in the `en.json`? I don't see it in your PR.

Are there any other strings that are also missing?